apps/web/src/pages/ItTrainingFundPage/ItTrainingFundPage.tsx

```diff
@@ -0,0 +1,578 @@
+                        <Link
+                          mode="text"
+                          data-h2-font-weight="base(bold)"
+                          color="secondary"
+                          external
+                          href={signUpUrl[locale]}
+                        >
+                          {intl.formatMessage({
+                            defaultMessage:
+                              "Sign up for updates<hidden>about instructor-led classes and bootcamps</hidden>",
+                            id: "uuE/zT",
+                            description: "A link to sign up for updates",
+                          })}
+                        </Link>
+                      </div>
+                    </div>
+                  </div>
+                </CardBasic>
+
+                <CardBasic
+                  data-h2-overflow="base(hidden)"
+                  data-h2-padding="base(0)"
+                  data-h2-display="base(flex)"
+                  data-h2-flex-direction="base(column)"
+                >
+                  <div
+                    data-h2-display="base(block) base:children[>span](block)"
+                    data-h2-padding="base(x1)"
+                    data-h2-background-color="base(gray.darkest) base:dark(foreground.shade)"
+                    data-h2-color="base:all(white)"
+                  >
+                    <span data-h2-font-size="base(h6, 1)">
+                      {intl.formatMessage({
+                        defaultMessage: "3. Certification exam vouchers",
+                        id: "vfXTEA",
+                        description: "Title for cert exam card",
+                      })}
+                    </span>
+                  </div>
+                  <div
+                    data-h2-display="base(flex)"
+                    data-h2-flex-direction="base(column)"
+                    data-h2-justify-content="base(space-between)"
+                    data-h2-flex-grow="base(1)"
+                  >
+                    <img
+                      src={image3}
+                      alt=""
+                      data-h2-display="base(none) p-tablet(block)"
+                    />
+
+                    <div
+                      data-h2-padding="base(x1)"
+                      data-h2-flex-grow="base(1)"
+                      data-h2-display="base(flex)"
+                      data-h2-flex-direction="base(column)"
+                      data-h2-gap="base(x1)"
+                    >
+                      <p>
+                        <strong>
+                          {intl.formatMessage({
+                            defaultMessage: "Coming in winter 2024-25",
+                            id: "h7f4Om",
+                            description:
+                              "Statement that something will be available in the future",
+                          })}
+                        </strong>
+                      </p>
```

Comment:
Same thing here with the font weight.

Reply:
Good catch!  I've switched out all the strong tags for font-weights.
[remove strong tags](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11821/commits/82a5dc913b2efd9e4edd6790d4d3886c7584932e)

The inline strongs do get changed to font weight 700 by our intl provider.
packages/i18n/src/components/richTextElements.tsx